### PR TITLE
HPCC-19031 Special characters in docbook XML files

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>
@g-pan Please review 
Test build at 10.240.32.243/view/Docs/job/JimDocBuild7/CE=ubuntu-14.04-amd64-1/

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by JamesDeFabia** [comment]
@g-pan please review

File: docs/EN_US/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml [hunk 1]
```diff
@@ -362,7 +362,7 @@
     <programlisting>__int64 rtlSumCharLen(unsigned len, char a[][20]) {
     __int64 sumtrimedlen = 0;
        for(unsigned i = 0; i &lt; len; i++) {
-          for(int j = 20-1; j &gt;= 0; j—) {
+          for(int j = 20-1; j &gt;= 0; j--) {
```
Comment (g-pan) [inline docs/EN_US/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml]: Check this change. In my opinion, this change is correcting an error.

File: docs/EN_US/HPCCMonitoring/HPCCMonitoringAndReporting.xml [hunk 2]
```diff
@@ -824,7 +824,7 @@
 
         <para><programlisting>./hpcc-nagios-tools -ec -env /etc/HPCCSystems/environment.xml  \
 -enable_host_notify -enable_service_notify -set_url localhost/nagios3 \
--disable_check_all_disks –out /etc/nagios3/conf.d/hpcc_notifications.cfg
+-disable_check_all_disks --out /etc/nagios3/conf.d/hpcc_notifications.cfg
```
Comment (g-pan) [inline docs/EN_US/HPCCMonitoring/HPCCMonitoringAndReporting.xml]: should this be -out or --out

File: docs/EN_US/HPCCClientTools/CT_Mods/ECLCC.xml [hunk 3]
```diff
@@ -124,13 +124,13 @@
...
       from a specified input file. It compiles the code into an executable
-      program (Such as an ‘EXE’ file in Windows). The resulting program, when
+      program (Such as an 'EXE' file in Windows). The resulting program, when
       executed, runs the job, writing any output to standard output.
       Alternatively, you could redirect the output to a file or pipe into
       another process. With the ECL Compiler, you do not need a supercomputer
       cluster to develop and run ECL code.</para>
 
-      <para>Running the ECL Compiler without any options (or specifying –help)
+      <para>Running the ECL Compiler without any options (or specifying --help)
```
Comment (g-pan) [inline docs/EN_US/HPCCClientTools/CT_Mods/ECLCC.xml]: please verify this change is desired.

**Review by g-pan — COMMENTED** [review]
Couple questions, comments inline.

File: docs/EN_US/HPCCClientTools/CT_Mods/ECLCC.xml (hunk 3 above)
Comment (JamesDeFabia) [reply docs/EN_US/HPCCClientTools/CT_Mods/ECLCC.xml]: either --help or -help works, IMHO, --help is preferred

File: docs/EN_US/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml (hunk 1 above)
Comment (JamesDeFabia) [reply docs/EN_US/ECLLanguageReference/ECLR_mods/ExtrSvcs-ExternalServicesImpl.xml]: correct

**Comment by JamesDeFabia** [comment]
@g-pan Edits made ... back to you

**Review by g-pan — APPROVED** [review]

**Comment by JamesDeFabia** [comment]
@richardkchapman This should be good to merge

**Pull request merged**